microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md

```diff
@@ -114,13 +114,11 @@ The main metric for measuring the performance of any animation is frames per sec
 
 1. Look at the **FPS** chart, shown below.  Whenever a red bar is displayed above **FPS**, it means that the framerate dropped so low that it's probably harming the user experience.  In general, the higher the green bar, the higher the FPS.
 
-   ![The FPS chart.](../media/evaluate-performance-performance-fps-chart.msft.png)
```

Comment:
Can you explain why you removed this image? The text before says "look at the FPS chart below", but there isn't a chart below anymore. So, it's a bit confusing. Shouldn't we just keep this image?

Reply:
From Reeva:  [10:06 AM] Reeza Ali
[9:47 AM] Reeza Ali
Jason Andrew (Wipro Designit Services, Inc.)If that's the case the work item needs to be revise and everything redone. I was just worker bee doing it, which is how it came up, but I can revise. Reeza Ali? Shall I do that?
Thanks for asking for clarification. 
 
I met with Zoher (1 month ago), and then Zoher and Rachel about 1-2 weeks ago. This was the guidance and changes they recommended we make to the doc. 
 
In terms of next steps: Let's button it up and send it to them for review. They can then guide us on what might need to change or not. 
 
[9:48 AM] Reeza Ali
Also, Rachel is planning to overhaul the performance docs in Dec. 